citus-rebalancer.spec

```diff
@@ -49,6 +49,9 @@ PATH=%{pginstdir}/bin:$PATH
 %{pginstdir}/share/extension/shard_rebalancer.control
 
 %changelog
+* Fri Dec 21 2018 - Hanefi Onaldi <hanefi@citusdata.com> 8.1.0.citus-1
+- Official release for 8.1.0
+
 * Tue Nov 06 2018 - Burak Velioglu <velioglub@citusdata.com> 8.0.0.citus-1
 - Official release for 8.0.0
 
```

Comment:
You should add steps similar to https://github.com/citusdata/packaging/commit/7589e5907ba67de8899e50f33005b203a509a484

Reply:
I added that part but I could not confirm if it is working properly now